### DISCUSSION TITLE
gm commands wow 3.3.5a azerothcore

### DISCUSSION THREAD

**Discussion by xBarselonx** [azerothcore/azerothcore-wotlk#19728] [General]
The GM command `.revive` does not resave the character.

**Comment by Underminus** [comment]
You will need to be a bit more specific.
 - Is this occurring on every playable character?
 - Is this only occurring when you are trying to revive yourself after dying? If YES, then does it work if you type " /g .revive "

GMs cannot speak in /say when they are dead. So you will always need to select a channel like /p or /g to enable any commands.

**Comment by Ramys** [comment]
> O comando `.revive` GM não salva novamente o personagem.

Too vague, needs to be more specific.
Follow the instructions to open a self-help form.
